### PR TITLE
fix(maxWidgetsAlert): Added a modal that serves as an alert message, …

### PR DESCRIPTION
**Pull request by hoganmaps**
…letting users know they have reached the maximum number of widgets on a dashboard. #607

### CODE REVIEW

**Comment by hoganmaps** [comment]
I checked this in ie9 through browserstack, chrome (latest) and firefox 24 locally. ie9 through browserstack worked, but adding the maximum number of widgets to ie9 is very memory intensive and not recommended.

File: src/OzoneConfig.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 window.OzoneConfig = {
-    'API_URL': 'https://www.owfgoss.org/ng/dev/mp',
+    'API_URL': 'https://localhost:7800/marketplace',
```
Comment (arw180) [inline src/OzoneConfig.js]: This may break some of our CI and/or dev-tools scripts. I'd revert this to be safe

File: src/OzoneConfig.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
     'CENTER_URL': 'https://ozone-development.github.io/ozp-center',
     'DEVELOPER_RESOURCES_URL': '#',
     'FEEDBACK_ADDRESS': 'mailto:person@address.com',
     'HUD_URL': 'https://ozone-development.github.io/ozp-hud',
-    'IWC_URL': 'https://ozone-development.github.io/iwc',
+    'IWC_URL': 'http://localhost/iwc-master',
```
Comment (arw180) [inline src/OzoneConfig.js]: This may break some of our CI and/or dev-tools scripts. I'd revert this to be safe

File: src/app/models.js [hunk 3]
```diff
@@ -298,98 +310,97 @@ models.factory('models', function($sce, $q, $log, $http, $window, useIwc,
...
+        var left = 75;
+        var width = 250;
+        var height = 250;
 
-      // extend array to be able to give us max value
-      Array.prototype.max = function() {
-        return Math.max.apply(null, this);
-      };
+        // extend array to be able to give us max value
+        Array.prototype.max = function() {
```
Comment (arw180) [inline src/app/models.js]: Is there a angular-friendly way of creating a single file where all such modifications to things like `Array.prototype` can be placed?

File: src/app/models.js [hunk 4]
```diff
@@ -298,98 +310,97 @@ models.factory('models', function($sce, $q, $log, $http, $window, useIwc,
...
-        }
-      };
+        };
 
-      dashboard.frames.push(newApp);
-      this.saveDashboard(dashboard);
-      return newApp;
+        dashboard.frames.push(newApp);
+        this.saveDashboard(dashboard);
+        console.log(newApp);
```
Comment (arw180) [inline src/app/models.js]: use `$log` instead

**Comment by SOAP1914** [comment]
I guess I'll wait until after the small changes above are done?

**Comment by SOAP1914** [comment]
:+1:

**Comment by arw180** [comment]
:+1:

**Pull request merged**